mila_datamodules/cli/prepare_torchvision.py

```diff
@@ -23,15 +24,27 @@
 current_cluster = Cluster.current_or_error()
 
 
-class PrepareVisionDataset(Generic[VD, P]):
+class AbstractPrepareVisionDataset(Generic[VD, P]):
```

Comment:
Why the need for an additional abstract base class? I'm not a big fan of those. Having "abstract" in the name is also a bit ugly IMO.

Reply:
I agree I don't like the name _Abstract_ but it is a known concept I believe so people understand from reading the name what it is. But I will use what you suggested, it's nicer to read